apps/omg_watcher_rpc/priv/swagger/informational_api_specs/swagger.yaml

```
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   version: '1.0.0'
-  title: Watcher's Informational API
+  title: Watchers Informational API
```

```suggestion
  title: Watcher Informational API
```

Just make it simple and drop the 's'